tasks.php

```diff
@@ -1160,6 +1160,9 @@ function TaskForm($task)
         $tasks_status_array = [1 => "New"];
     }
 
+    // Don't want to permit setting status to 'Closed' when creating/editing a task
+    unset($tasks_status_array[14]);
```

Comment:
I realize it is the way it is currently, but I'm not a huge fan of [unnamed numerical constants](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)
It makes it hard to know what the code is doing instead of being self documented.

Reply:
I've done some work along those lines @chrismiceli - does that look better?

Reply:
Must better in my opinion, thank you!